bigbluebutton-html5/imports/ui/components/connection-manager/component.tsx

```diff
@@ -73,7 +73,13 @@ const errorLink = onError(({ graphQLErrors, networkError }) => {
   }
 
   if (networkError) {
-    connectionStatus.setSubscriptionFailed(true);
+    if (
+      networkError?.cause
+      // Mutations doesn't have extensions, only subscriptions
+      // @ts-ignore - extensions is not in the type definition
+      && networkError.cause.extensions) {
```

Comment:
I don't think it is a good solution as the property `extensions` is just part of the implementation of Hasura. But it's not required by Graphql specification.

In my opinion looking for `graphql actions request failed` in the error message is a better approach.